### PR TITLE
doc: release-notes-latest: Summarize other changes from the upmerge

### PR DESCRIPTION
**Pull request by carlescufi**
Summarize all uncategorized items in their respective sections.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

**Review by ru-fu — CHANGES_REQUESTED** [review]
Will add a commit.

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -209,8 +228,54 @@ The following list summarizes the most important changes inherited from upstream
...
+
+* Logging
+
+  * Added STP transport and raw data output support for systrace
+
+* Modules
+
+  * mbedtls: Introduced MBEDTLS_MEMORY_DEBUG option
+  * lvgl: Updated LVGL to v7.6.1
+  * openamp: Updated libmetal and openamp to v2020.10
```
Comment (anangl) [inline doc/nrf/releases/release-notes-latest.rst:281]: I think it should be also mentioned that nrfx in hal_nordic was updated to version 2.4.0.

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -209,8 +229,50 @@ The following list summarizes the most important changes inherited from upstream
...
     * Fixed handling of zero-length packets (ZLP) in the Nordic Semiconductor USB Device Controller driver (usb_dc_nrfx).
     * Fixed initialization of the workqueue in the usb_dc_nrfx driver, to prevent fatal errors when the driver is reattached.
     * Fixed handling of the SUSPEND and RESUME events in the Bluetooth classes.
+
+* Build system:
+
+  * Ensured that shields can be placed in other BOARD_ROOT folders.
+  * Added basic support for Clang 10 with x86.
+
+* System:
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:247]: Is that was Sys stands for?

File: doc/nrf/releases/release-notes-latest.rst [hunk 3]
```diff
@@ -209,8 +229,50 @@ The following list summarizes the most important changes inherited from upstream
...
+* Modules:
+
+  * Introduced a :option:`CONFIG_MBEDTLS_MEMORY_DEBUG` option for mbedtls.
+  * Updated LVGL to v7.6.1.
+  * Updated libmetal and openamp to v2020.10.
+
+* Other:
+
+  * Added initial LoRaWAN support.
+  * Updated west to fail if a HEX file has UICR data and ``--erase`` was not specified.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:278]: Is that what you were trying to say?

**Review by ru-fu — COMMENTED** [review]
Added a commit - please check and squash, if okay.

File: doc/nrf/releases/release-notes-latest.rst (hunk 3 above)
Comment (mbolivar-nordic) [reply doc/nrf/releases/release-notes-latest.rst:278]: The "nrfjprog" part is important. How about "Updated `west flash` support for `nrfjprog` to fail if the HEX file has UICR data and `--erase` was not specified." or so?

Comment (ru-fu) [reply doc/nrf/releases/release-notes-latest.rst:278]: OK, sounds good.

File: doc/nrf/releases/release-notes-latest.rst (hunk 1 above)
Comment (ioannisg) [reply doc/nrf/releases/release-notes-latest.rst]: Doing that @anangl thanks

File: doc/nrf/releases/release-notes-latest.rst (hunk 2 above)
Comment (ioannisg) [reply doc/nrf/releases/release-notes-latest.rst:247]: Yeap, think so.

**Review by ioannisg — APPROVED** [review]
lgtm

File: doc/nrf/releases/release-notes-latest.rst [hunk 4]
```diff
@@ -387,3 +407,44 @@ The following list summarizes the most important changes inherited from upstream
...
+  * Updated LVGL to v7.6.1.
+  * Updated libmetal and openamp to v2020.10.
+  * Updated nrfx in hal-nordic to version 2.4.0.
+  * Updated the Trusted Firmware-M (TF-M) module to include support for the nRF5340 and nRF9160 platforms.
+
+
+* Other:
+
+  * Added initial LoRaWAN support.
+  * Updated west to fail if a HEX file has UICR data and ``--erase`` was not specified.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:450]:
```suggestion
  * Updated ``west flash`` support for ``nrfjprog`` to fail if a HEX file has UICR data and ``--erase`` was not specified.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 5]
```diff
@@ -362,10 +386,6 @@ The following list summarizes the most important changes inherited from upstream
     * Added an API for resetting a node (:c:func:`bt_mesh_cfg_node_reset`).
     * Added an API for setting network transmit parameters (:c:func:`bt_mesh_cfg_net_transmit_set`).
 
-* Trusted Firmware-M:
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:365]: Has that moved?

**Review by anangl — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst (hunk 5 above)
Comment (ioannisg) [reply doc/nrf/releases/release-notes-latest.rst:365]: yes, together with the other zephyr modules.

**Pull request merged**